src/components/Onboarding/content.js

```diff
@@ -1,29 +1,34 @@
 import React from 'react'
+
 import large1 from './highlights/assets/rinkeby/large/1.png'
 import large2 from './highlights/assets/rinkeby/large/2.png'
 import large3 from './highlights/assets/rinkeby/large/3.png'
 import large4 from './highlights/assets/rinkeby/large/4.png'
 import large5 from './highlights/assets/rinkeby/large/5.png'
 
-import small1 from './highlights/assets/rinkeby/small/1.png'
-// import small2 from './highlights/assets/rinkeby/small/2.png'
-import small3 from './highlights/assets/rinkeby/small/3.png'
-import small4 from './highlights/assets/rinkeby/small/4.png'
-import small5 from './highlights/assets/rinkeby/small/5.png'
+const highlights = {
+  heading: 'Aragon Court',
+  defaultVisualColor: '#FF9780',
+}
 
-const highlights = {}
-highlights.latest = [
+highlights.rinkeby = [
   {
     title: {
       small: null,
       large: 'Welcome, juror',
     },
     description: {
       small: null,
-      large: `This is a preview release of the official Court Dashboard that will launch on Feb 10th. We'd love for you to explore it and tell us what you think!`,
+      large: (
+        <span>
+          This is a preview release of the official Court Dashboard that will
+          launch on Feb 10th. We'd love for you to explore it and tell us what
+          you think!
+        </span>
+      ),
     },
     visual: {
-      small: small1,
+      small: null,
```

Comment:
why are we not using the small images anymore?

Reply:
This is meant to be used for alternative images on the mobile layout, but in this case they were the same (I think? I need to check again :s), so I thought we might as well keep the large ones only.

Reply:
Yes:

- Image 1: [small](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/small/1.png) and [large](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/large/1.png)
- Image 2: (small was missing)
- Image 3: [small](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/small/3.png) and [large](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/large/3.png)
- Image 4: [small](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/small/4.png) and [large](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/large/4.png)
- Image 5: [small](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/small/5.png) and [large](https://github.com/aragon/court-dashboard/blob/229953d4cab4e62d92d5f2eff0ad3f5860513602/src/components/Onboarding/highlights/assets/rinkeby/large/5.png)

Reply:
Yeah i added those small ones because patty add separated images for mobile and for desktop, but maybe the only change is the size of the image